src/pages/event/eventDetail/eventDetail.tsx

```diff
@@ -0,0 +1,177 @@
+                  <>
+                    <div className="font-bold">Location</div>
+                    <div className="lg:w-4/6">{location}</div>
+                  </>
+                )}
+                <div className="flex flex-col pt-4">
+                  {videoLink && (
+                    <div className="w-full sm:w-auto mb-2 sm:mb-0">
+                      <a
+                        className="text-lg text-cerulean-200 font-medium hover:text-cerulean-300 inline-block pr-4 cursor-pointer"
+                        href={videoLink}
+                        target="_blank"
+                        rel="noopener noreferrer"
+                        data-testid="watch-link"
+                      >
+                        <div className="flex">
+                          <div className="w-auto">
+                            <PlayCircle />
+                          </div>
+                          <div className="flex-grow px-2">Watch Event</div>
+                        </div>
+                      </a>
+                    </div>
+                  )}
+                  <div className="w-full sm:w-auto mb-2 sm:mb-0 mt-4">
+                    <a
+                      className="text-lg text-cerulean-200 font-medium hover:text-cerulean-300 inline-block pr-4"
+                      href={link}
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      data-testid="event-link"
+                    >
+                      <div className="flex">
+                        <div className="w-auto">
+                          <ExternalLink />
+                        </div>
+                        <div className="flex-grow px-2">Event Link</div>
+                      </div>
+                    </a>
+                  </div>
+
+                  {slides && (
+                    <div className="w-full sm:w-auto mb-2 sm:mb-0 mt-4">
+                      <a
+                        className="text-lg text-cerulean-200 font-medium hover:text-cerulean-300 inline-block pr-4 cursor-pointer"
+                        href={slides}
+                        target="_blank"
+                        rel="noopener noreferrer"
+                        data-testid="event-slides"
+                      >
+                        <div className="flex">
+                          <div className="w-auto">
+                            <ExternalLink />
+                          </div>
+                          <div className="flex-grow px-2">Event Slides</div>
+                        </div>
+                      </a>
+                    </div>
+                  )}
+                  {downloadableMediaContent !== undefined &&
+                    downloadableMediaContent.map((downloadableContent, index) => {
+                      if (!downloadableContent) return null;
+                      const fileName = downloadableContent.match(/[A-Za-z0-9_\-\.]+\.[A-Za-z0-9]+$/)?.shift();
```

Comment:
what's this? to remove ext? any other better way

Reply:
actually this is to select the filename
i.e. "/static/uploads/tradetrust-newsletter-1.pdf" --> "tradetrust-newsletter-1.pdf"

Reply:
is it regex ugly?
LOL